google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/TraceLoggingEnhancer.java

```diff
@@ -35,7 +35,7 @@ public TraceLoggingEnhancer(String prefix) {
   /**
    * Set the Trace ID associated with any logging done by the current thread.
    *
-   * @param id The traceID
+   * @param id The traceID, in the form projects/[PROJECT_ID]/traces/[TRACE_ID]
```

Comment:
Have you considered also getting the project ID as an argument and building the `projects/[PROJECT_ID]/traces/[TRACE_ID]` format from there?

Reply:
The convention for that in this repo would be to create a `TraceName` class that implements `ResourceName` and has a Builder pattern (or static constructor) and a formatting `toString()` method.

Reply:
OK, LGTM then